Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -228,7 +228,7 @@ static inline ASPublicKeyCredentialResidentKeyPreference toASCResidentKeyPrefere
         requestTypes &= ~ASCCredentialRequestTypePlatformPublicKeyRegistration;
 
     auto requestContext = adoptNS([allocASCCredentialRequestContextInstance() initWithRequestTypes:requestTypes]);
-    [requestContext setRelyingPartyIdentifier:options.rp.id];
+    [requestContext setRelyingPartyIdentifier:*options.rp.id];
```

Comment:
Since `options.rp.id` may now be `std::nullopt`, do we need to do any validation here? Maybe at least ASSERT so we catch cases where a null opt got through?